modules/placeholder.js

```diff
@@ -0,0 +1,67 @@
+import Emitter from '../core/emitter';
+import Module from '../core/module';
+
+const className = 'ql-placeholder';
+
+class PlaceHolder extends Module {
```

Comment:
In some places the word placeholder is treated as two words (like here) and others it's treated as one (like in quill.js). I think placeholder is a real word so one is more correct but either way this should be consistently treated as one (placeholder, Placeholder, placeholderText) or two (placeHolder, PlaceHolder, placeHolderText).